src/Microsoft.AspNetCore.Blazor.Build/targets/Blazor.MonoRuntime.targets

```diff
@@ -0,0 +1,305 @@
+    </ItemGroup>
+
+    <!-- 
+    Write the list of files in /obj/debug/netstandard2.0/blazor/blazor/linker/ into 
+    /obj/debug/netstandard2.0/blazor/blazor/linker.marker.txt
+    -->
+    <WriteLinesToFile File="$(BlazorIntermediateLinkerResultFilePath)" Lines="@(_BlazorLinkerOutput)" Overwrite="true" />
+
+    <!-- Add /obj/debug/netstandard2.0/blazor/blazor/linker.marker.txt to the list of written files. -->
+    <ItemGroup>
+      <FileWrites Include="$(BlazorIntermediateLinkerResultFilePath)" />
+    </ItemGroup>
+  </Target>
+
+  <Target 
+    Name="CollectLinkerOutputs"
+    Condition="'$(BlazorShouldRunMonoLinker)' == 'true'"
+    DependsOnTargets="RunMonoLinker">
+    <!--
+    Read the outputs from the linker (from this run or a previous run) and set them in an item group for
+    later use.
+    -->
+    <ReadLinesFromFile File="$(BlazorIntermediateLinkerResultFilePath)">
+      <Output TaskParameter="Lines" ItemName="_OptimizedFiles"/>
+    </ReadLinesFromFile>
+    
+    <ItemGroup>
+      <BlazorItemSpec Include="@(_OptimizedFiles)">
+        <TargetOutputPath>$(ProjectDir)$(OutputPath)$(BaseBlazorRuntimeBinOutputPath)%(FileName)%(Extension)</TargetOutputPath>
+        <Type>Assembly</Type>
+        <PrimaryOutput Condition="'%(FileName)' == @(IntermediateAssembly->'%(FileName)')">true</PrimaryOutput>
+      </BlazorItemSpec>    
+      <FileWrites Include="@(BlazorItemSpec->WithMetadataValue('Type','Assembly')->'%(TargetOutputPath)')" />
+    </ItemGroup>
+  
+  </Target>
+
+  <Target
+    Name="ResolveRuntimeDependencies"
+    Condition="'$(BlazorShouldRunMonoLinker)' == 'false'"
+    Inputs="@(IntermediateAssembly);
+      @(ReferenceCopyLocalPaths->WithMetadataValue('Extension','.dll'));
+      @(BlazorLinkerDescriptor)"
+    Outputs="@(BlazorResolvedAssembliesOutput)"
+  >
+    <ItemGroup>
+      <_DependenciesParameter Include="@(ReferenceCopyLocalPaths->WithMetadataValue('Extension','.dll')->'--reference &quot;%(FullPath)&quot;')" />
+    </ItemGroup>
+    <PropertyGroup>
+      <_BclParameter>--base-class-library &quot;$(MonoBaseClassLibraryPath)&quot; --base-class-library &quot;$(MonoBaseClassLibraryFacadesPath)&quot;</_BclParameter>
+    </PropertyGroup>
+
+    <Exec Command="$(BlazorBuildExe) resolve-dependencies &quot;@(IntermediateAssembly->'%(FullPath)')&quot; @(_DependenciesParameter, ' ') $(_BclParameter) --output &quot;$(BlazorResolvedAssembliesOutputPath)&quot;" />
+
+  </Target>
+  
+  <Target
+    Name="GetBaseClassLibrary"
+    DependsOnTargets="ResolveRuntimeDependencies"
+    Condition="'$(BlazorShouldRunMonoLinker)' == 'false'">
+
+    <!--
+    At this point we have decided not to run the linker and instead to just copy the BCL from
+    the nuget package into the _framework/_bin folder. The only thing we need to do here is
+    collect the list of items that will go into _framework/_bin. This list is composed of:
+    * All the mono BCL files.
```

Comment:
That's not the case any more, is it?

Reply:
Yeah, I need to do a bit of cleanup, but I'm trying to get this in and do that later